src-gui/src/renderer/components/modal/introduction/slides/Slide00_FiatPricePreference.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, Typography, Paper, Button } from '@material-ui/core'
+import CardSelectionGroup from 'renderer/components/inputs/CardSelection/CardSelectionGroup'
+import CardSelectionOption from 'renderer/components/inputs/CardSelection/CardSelectionOption'
+import logo from '../../../../../assets/unstoppableSwapLogo.png'
```

Comment:
We try to avoid relative paths going up in the folder tree this far.

You should be able to replace this with: `assets/unstoppableSwapLogo.png`